bentoml/_internal/bento/docker/entrypoint.sh

```diff
@@ -3,27 +3,51 @@ set -Eeuo pipefail
 
 # check to see if this file is being run or sourced from another script
 _is_sourced() {
-  # https://unix.stackexchange.com/a/215279
-  [ "${#FUNCNAME[@]}" -ge 2 ] \
-    && [ "${FUNCNAME[0]}" = '_is_sourced' ] \
-    && [ "${FUNCNAME[1]}" = 'source' ]
+	# https://unix.stackexchange.com/a/215279
+	[ "${#FUNCNAME[@]}" -ge 2 ] &&
+		[ "${FUNCNAME[0]}" = '_is_sourced' ] &&
+		[ "${FUNCNAME[1]}" = 'source' ]
 }
 
 _main() {
-  # if first arg looks like a flag, assume we want to start bentoml YataiService
-  if [ "${1:0:1}" = '-' ]; then
-    set -- bentoml serve --production "$@" $BENTO_PATH
-  fi
+	# Overide the BENTOML_PORT if PORT env var is present. Used for Heroku
+	if [[ -v PORT ]]; then
+		echo "\\$PORT is set! Overiding \\$BENTOML_PORT with \\$PORT ($PORT)"
+		export BENTOML_PORT=$PORT
+	fi
 
-  # Overide the BENTOML_PORT if PORT env var is present. Used for Heroku
-  if [[ -v PORT ]]; then
-    echo "\$PORT is set! Overiding \$BENTOML_PORT with \$PORT ($PORT)"
-    export BENTOML_PORT=$PORT
-  fi
+	# if file /etc/arg_mamba_user exists, then we are using micromamba-docker
+	if [[ -f /etc/arg_mamba_user ]]; then
```

Comment:
isn't this something we can do in the docker image template?

Reply:
Well I tried to put it into the file as {% raw %}, but jinja refuses to render `$`

Reply:
don't we have `$` everywhere in the templates?